### PR TITLE
Contact avatar mediatype

### PR DESCRIPTION
**Pull request by ksmurchison**
Better handling of missing mediatype on vCard PHOTO property

### CODE REVIEW

File: imap/vcard_support.c [hunk 1]
```diff
@@ -126,6 +126,21 @@ EXPORTED struct vparse_card *record_to_vcard(struct mailbox *mailbox,
     return vcard;
 }
 
+static const struct image_signature {
+    const uint8_t magic[8];
+    size_t magiclen;
+    const char *mediatype;
+} image_signatures[] = {
```
Comment (rsto) [inline imap/vcard_support.c:133]: Wikipedia lists a couple of additional magic bytes for the respective types:

# JPG
```
FF D8 FF DB
FF D8 FF E0 00 10 4A 46
49 46 00 01
FF D8 FF EE
FF D8 FF E1 ?? ?? 45 78
69 66 00 00
```

where `??` is the data size.

# Google WebP format (`image/webp`)
```
52 49 46 46 ?? ?? ?? ??
57 45 42 50
```

where `??` is the file size.

**Review by rsto — APPROVED** [review]
Nice! I found some additional magic markers that we might want to support. In any case, this looks good to me.

**Comment by ksmurchison** [comment]
Added support for 'split' magic numbers and a couple more formats.  I may add support for SVG as well

**Review by rsto — APPROVED** [review]
Looks good to me.

**Review by rsto — APPROVED** [review]

**Pull request merged**